src/elements/Typography/Typography.sass

```diff
@@ -0,0 +1,15 @@
+.typography h1
+  font-weight: 400
+  font-size: 36px
+  margin-bottom: 24px
+  line-height: 1.4
+  color: var(--global-black)
+  font-family: var(--main-font-family)
+
+.typography h2
+  font-weight: 400
```

Comment:
we don't need to repeat base styles for each variant

Reply:
reference: https://dev.to/dcwither/typography-separating-style-from-semantics-n3f
try to not copy implementation, try to understand how to separate and don't repeat styles